@navikt/core/react/src/util/Slot.tsx

```diff
@@ -0,0 +1,85 @@
+// https://github.com/radix-ui/primitives/blob/main/packages/react/slot/src/Slot.tsx
+import * as React from "react";
+import mergeRefs from "./mergeRefs";
+
+interface SlotProps extends React.HTMLAttributes<HTMLElement> {
+  children?: React.ReactNode;
+}
+
+export const Slot = React.forwardRef<HTMLElement, SlotProps>(
+  (props, forwardedRef) => {
+    const { children, ...slotProps } = props;
+
+    return (
+      <SlotClone {...slotProps} ref={forwardedRef}>
+        {children}
+      </SlotClone>
+    );
+  }
+);
+
+interface SlotCloneProps {
+  children: React.ReactNode;
+}
+
+const SlotClone = React.forwardRef<any, SlotCloneProps>(
```

Comment:
Kan vi skrive `HTMLElement` i stedet for `any` tro? 🤔
```suggestion
const SlotClone = React.forwardRef<HTMLElement, SlotCloneProps>(
```

Reply:
Fikset 👍